src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,69 @@
+package duke.parser;
+
+import duke.exception.DukeException;
+
+/**
+ * A parser for reading user input
+ */
+public class Parser {
+    private String msg;
+
+    /**
+     * Constructor for <code>Parser</code>.
+     *
+     * @param msg a string of input from user
+     */
+    public Parser(String msg) {
+        this.msg = msg;
+    }
+
+    /**
+     * Process msg and return the request (command) of the input.
```

Comment:
Should `msg` be a standard word instead of a "texting-style" spelling as it is part of the javadoc and is meant for users to read? 
I noticed several other places with this issue too.